packages/core/forms/src/components/fields/FieldObjectAdvanced.vue

```diff
@@ -216,9 +219,10 @@ export default {
         Array.isArray(this.subSchema.fields) &&
         this.subSchema.fields.length === 1 &&
         this.subSchema.fields[0].type === 'array'
-      const type = this.subSchema ? valueIsArray ? [''] : {} : ''
+      const defaultValue = this.schema.values?.default ?? ''
```

Comment:
If value's type is not string, would it cause problem to set default value as ''?

Reply:
Updated.